taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+import re
+
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = "__all__"
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "The license number must contain 8 characters."
+            )
+
+        if not re.match(r"^[A-Z]{3}", license_number):
+            raise forms.ValidationError(
+                "The first 3 characters must be uppercase letters."
+            )
```

Comment:
The regular expression `r"^[A-Z]{3}"` used here only checks if the first three characters are uppercase letters, but it doesn't ensure that they are the only characters in the first part of the license number. Consider using `r"^[A-Z]{3}\d{5}$"` to ensure the entire license number format is validated in one go.